format/simple.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable prefer-object-spread */
+const { format } = require('logform');
+const { MESSAGE } = require('triple-beam');
+const jsonStringify = require('fast-safe-stringify');
+
+/*
+ * function simple (info, key)
+ * An adjusted version of logform's simple() format.
+ * Contains a different format that includes the timestamp.
+ * key is the metadata key, if exists.
+ *
+ * src: https://github.com/winstonjs/logform/blob/master/simple.js
+ */
+module.exports = format((info, key = '') => {
+  const stringifiedRest = jsonStringify(Object.assign({}, info, {
+    level: undefined,
+    message: undefined,
+    timestamp: undefined,
+    splat: undefined,
+  }));
+
+  if (stringifiedRest !== `{"${key}":{}}`) {
```

Comment:
True.

Reply:
Just FYI, this was taken from `simple` format.

Reply:
Oh dear god, I have no idea what's going on here. Why assign stuff to info? Why the ultra-weird emptiness check? Why do we even need/want this?

Reply:
OK, I get where we use it now. I still can't fathom why it's implemented this way, but as I understand it it was like this before... yuck.

Reply:
> Hm, this row made me think twice before I understood why is that.

I agree, it's not trivial line and pretty ugly.

> Why not just check that the `info` has something under `key`?

Because it's not enough. Meta can be under the meta key OR root of info. 
We essentially need the difference set of two objects (by keys). We can always iterate on keys and stuff like that, I'm not sure it would make it more readable given that we need to use the JSON output anyway.

Reply:
I've updated this file. Please check again.